.buildkite/scripts/integration-tests.ps1

```diff
@@ -37,7 +39,12 @@ $outputJSON = "build/${fully_qualified_group_name}.integration.out.json"
 try {
     Get-Ess-Stack -StackVersion $PACKAGE_VERSION
     Write-Output "~~~ Running integration test group: $GROUP_NAME as user: $env:USERNAME"
+<<<<<<< HEAD
     gotestsum --no-color -f standard-quiet --junitfile "${outputXML}" --jsonfile "${outputJSON}" -- -tags=integration -shuffle=on -timeout=2h0m0s "github.com/elastic/elastic-agent/testing/integration" -v -args "-integration.groups=$GROUP_NAME" "-integration.sudo=$TEST_SUDO"
+=======
+    & gotestsum --no-color -f standard-quiet --junitfile-hide-skipped-tests --junitfile "${outputXML}" --jsonfile "${outputJSON}" -- -tags=integration -shuffle=on -timeout=2h0m0s "github.com/elastic/elastic-agent/testing/integration" -v -args "-integration.groups=$GROUP_NAME" "-integration.sudo=$TEST_SUDO"
+    $TestsExitCode = $LASTEXITCODE
+>>>>>>> 05289da6b (ci(test): skip skipped tests from the Test JUnit output (#8084))
```

Comment:
IIUC, https://github.com/elastic/elastic-agent/pull/6662 was not backported to `9.0`

```suggestion
    & gotestsum --no-color -f standard-quiet --junitfile-hide-skipped-tests --junitfile "${outputXML}" --jsonfile "${outputJSON}" -- -tags=integration -shuffle=on -timeout=2h0m0s "github.com/elastic/elastic-agent/testing/integration" -v -args "-integration.groups=$GROUP_NAME" "-integration.sudo=$TEST_SUDO"
    $TestsExitCode = $LASTEXITCODE
```

Reply:
https://github.com/elastic/elastic-agent/pull/8170 is needed